### ISSUE TITLE
Hyperparameters for pre-training

### ISSUE THREAD

**Issue by Humanoid-z** [chao1224/MoleculeSTM#4]
Hi,
First of all thank you for your work! I noticed that the default hyperparameters for the pre-training scripts seem to be inconsistent with the paper, or some are not mentioned in the paper. 
Do you confirm that these are the hyperparameters that were ultimately chosen? If not is it possible to provide the hyperparameters for pre-training?
Thank you.

**Comment by chao1224** [comment]
Hi @SNIKCHS,

Thank you for your interest in our work.
The hyperparameters reported in the paper are what we ran, and we will let you know once we fixed this.

**Comment by Greay83** [comment]
@SNIKCHS Hello, I have been having trouble configuring the project runtime environment, especially the version matching of pytorch, pyg, cuda. Could you please provide some details on the configuration of your running environment?
Thank you very much!

**Comment by XingyuLu206** [comment]
Dear author, I am a little confused, in the paper the learning rate for the chemical structure branch is {1e-5, 3e-5} for pre-training, but the script in the code sets the value as 1e-4, could you please let me know which one should I choose to reproduce your experiment results?

**Comment by chao1224** [comment]
Hi @Nipers ,

Sorry for the confusion. Please follow the paper appendix. Now the configs (under the `scripts` folder) are only for demo, and I will update them in the next version.
